### DISCUSSION TITLE
Processing of academic proof takes forever

### DISCUSSION THREAD

**Discussion by Apfelstrudel123** [community/community#101758] [GitHub Education]
### Hi everyone,

I'm, having trouble extending my GitHub Student benefits. When I upload the new proof of my academic status and hit "Process my application" the button goes to "Processing information" but after that nothing happens. I've left the tab open for a few hours and tried it from different devices everytime the same result. Has anyone else experienced this or knows what might cause the issue?

Thanks!

**Comment by liamli99** [comment]
I have experienced the same thing. I wait for seven business days and receive no response, so that I directly take the picture of my student ID card and submit it, after two days, I receive the confirmation email.
